### DISCUSSION TITLE
Why is a catbox link required for the plugin listing?

### DISCUSSION THREAD

**Discussion by Rustly** [Pryaxis/TShock#2026] [General]
I'll admit I'm lazy. Is there a specific reason why you specify a catbox link versus just say, linking the location of build/release page of the repo?

**Comment by hakusaro** [comment]
The whole point of “approval” is that we’re checking the plugins people upload for obvious signs of malware or tampering. If you just link to some random binary, you might swap it out with bad things, it might not be up for long. As I’ve repeatedly said, I have a financial interest in Catbox and making sure it doesn’t go down.

**Comment by moisterrific** [comment]
I don’t mind Catbox, btw for any plugin devs reading this: you might wanna upload it there as a zipped folder instead of only the binary cuz Catbox renames the file to some random letters. 

Just makes it a little nicer for whoever is installing the plugin.
